### PR TITLE
Normalize headings and document structure.

### PR DESCRIPTION
**Pull request by jpmedley**
An equivalent PR is being made in sw-toolkit so that both have roughly similar section titles and order. A copy edit of the content will be in a later PR.

### CODE REVIEW

**Comment by jeffposnick** [comment]
Thanks for taking this on, @jpmedley! It looks like the **Command-line interface** section disappeared as part of this reshuffling. Could you restore that content?

**Comment by jpmedley** [comment]
@jeffposnick It's still there. I felt it needed a more specific name. It's now called 'Using with Gulp and Grunt' and was moved higher on the page.

**Comment by jeffposnick** [comment]
The command-line interface is for folks who **don't** want to use `sw-precache` via Gulp or Grunt, so **Using with Gulp and Grunt** is not an appropriate section title. The first sentence of that section hopefully makes that clear.

File: README.md [hunk 1]
```diff
@@ -296,6 +302,20 @@ Thanks to [Sindre Sorhus](https://github.com/sindresorhus) and
 
 [Apache 2.0](https://github.com/googlechrome/sw-precache/blob/master/LICENSE) © 2015 Google Inc.
 
+Copyright 2015 Google, Inc.
+
+Licensed under the [Apache License, Version 2.0](https://github.com/googlechrome
+/sw-precache/blob/master/LICENSE)] (the "License"); you may not use this file
```
Comment (jeffposnick) [inline README.md]: The same issue called out in https://github.com/GoogleChrome/sw-toolbox/pull/51 applies here.

**Comment by jpmedley** [comment]
@jeffposnick PTAL

(Sorry about the command-line thing.)

**Comment by jeffposnick** [comment]
I'm not digging the `Options` section above the `Methods` section. There's a reference to `generate()`/`write()` within the `Options` preamble, but we don't actually describe what those methods are until after all the options are listed. I think the `Methods` need to come first so that folks understand how those `Options` are being used.

**Comment by jpmedley** [comment]
I'll take a closer look. I'm thinking about what the landscape looks like when we have 10 or 20 libraries. We want the sections to be ordered similarly.

**Comment by jpmedley** [comment]
@jeffposnick PTAL

File: README.md [hunk 2]
```diff
@@ -106,24 +110,49 @@ load). The sample service-worker-registration.js file
...
 
-### write(filePath, options, callback)
+#### write(filePath, options, callback)
 `write` is a helper method that calls `generate` and takes the resulting string content and
 writes it to disk, at `filePath`. It then invokes `callback(error)`.
 
 Since 2.2.0, `write()` also returns a [`Promise`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise).
 
-## Options
+### Options Parameter
```
Comment (jeffposnick) [inline README.md]: If "Options Parameter" is prefixed by `###` now, then shouldn't all the sub-items that follow it be prefixed with `####`? E.g. `### cacheId` should change to `#### cacheId`, etc.?

**Comment by jpmedley** [comment]
Done. Please merge.

**Pull request merged**